Document/0x05d-Testing-Data-Storage.md

```diff
@@ -62,6 +62,51 @@ root@hermes:/data/data/sg.vp.owasp_mobile.myfirstapp/shared_prefs # ls -la
 
 > Please note that `MODE_WORLD_READABLE` and `MODE_WORLD_WRITEABLE` were deprecated starting on API level 17. Although newer devices may not be affected by this, applications compiled with an `android:targetSdkVersion` value less than 17 may be affected if they run on an OS version that was released before Android 4.2 (API level 17).
 
+##### Shared Preferences (Encrypted)
```

Comment:
Please remove this section. For encrypted shared preferences we already have an issue and decided to not add it at the moment as the library is considered as alpha. https://github.com/OWASP/owasp-mstg/issues/1662

Reply:
Hi, I will remove this section, but the library is now a release candidate. I will add it as a comment to PR 1662. Thanks.

Reply:
Those are great news, please do so and pack the contents in a new PR. That'd definitely help us reviewing it properly.